lightstep/resource_metric_condition_test.go

```diff
@@ -216,7 +180,7 @@ resource "lightstep_metric_condition" "test" {
 					testAccCheckMetricConditionExists(resourceName, &condition),
 					resource.TestCheckResourceAttr(resourceName, "name", "Too many requests"),
 					resource.TestCheckResourceAttr(resourceName, "description", "A link to a playbook"),
-					resource.TestCheckResourceAttr(resourceName, "metric_query.0.timeseries_operator_input_window_ms", "3600000"),
+					resource.TestCheckResourceAttr(resourceName, "query.0.query_string", uqlQuery+"\n"),
```

Comment:
this trailing newline is probably a bug but 🤷 a pretty inoffensive one

Reply:
Hmm. I might be misinterpreting what you're saying, but the `+"\n"` is needed on all these tests since the `<<EOT` docstring syntax always adds a newline (which ends up being passed to the SaaS which does not do any trim or modification of the raw string) and the variable itself in this test code doesn't have that newline.